docs/architecture/adr-011-optimistic-blob-size-independent-inclusion-proofs-and-pfd-fraud-proofs.md

```diff
@@ -0,0 +1,173 @@
+
+Commitment inclusion proof size over one share PFB
+= subtree root + blue nodes to share + blue nodes to (`DataRoot`)  
+= NMT-Node size + log(k) \* NMT-Node size + log(k) \* MT-Node size  
+= 48 + log(k) \* 48 + log(k) \* 32  
+= 48 + 80 \* log(k)
+
+For two shares the best case is the same as for one share but the worst case includes twice the blue nodes to the row root. For 3 and 4 shares for a PFB transaction, the worst case proof size does not change significantly.
+
+Worst case commitment inclusion proof size over 2-4 share PFB
+= subtree roots + blue nodes to share + blue nodes to (`DataRoot`)  
+= 2 \* NMT-Node size + 2 \* log(k) \* NMT-Node size + log(k) \* MT-Node size  
+= 96 + log(k) \* 96 + log(k) \* 32  
+= 96 + 128 \* log(k)
+
+![Optimized Pfb Proofs](./assets/optimized-pfb-proofs.png)
+
+<!--- This does not need a fraud proof as it could be a validation rule that even light clients can check. This would require the light clients to know the sequencer set and whose turn it was. (not sure about this)
+--->
+The fraud proof for this would be to prove that the commitment of the PFB transaction does not equal the predicted commitment in the header. Therefore this is equivalent to a PFB transaction inclusion proof. This fraud proof would be optimistic as we would assume that the PFB commitment is correct. But realistically if the commitment over the PFB transaction is wrong then the PFB commitment is most likely wrong as well. Therefore the fraud poof would be a PFB Fraud Proof as described at the top.  
+If we do not have a PFB transaction that can be predicted, we also need to slash double signing of 2 valid PFB transactions in Celestia. This is required so we don't create a valid fraud proof over a valid commitment over the PFB transaction.
+
+The third optimization could be to SNARK the PFB Inclusion Proof to reduce the size even more.?
+
+## Result
+
+![PFB Inclusion Proof](./assets/pfb-proof-size-result.png)
+
+For normal-sized PFBs the Proof size will be worth it from blob size x to y depending on how many shares the PFB occupies.
+If the PFB is guaranteed to be in one share then it is always worth from blob size x.
+For Huge PFBs that span over 4 shares the PFB Proof will be worth it from blob size z.
+Detailed analysis of x,y and z will follow if the general concept makes sense.
+
+## Alternative Approaches
+
+The other way to prove blob inclusion is dependent on the blob size. A blob inclusion proof includes all the subtree roots of the blob and the Merkle proof of those subtree roots to the `DataRoot` of Celestia.
+
+## Detailed Design
+
+TODO
+
+## Status
+
+Proposed
+
+## Consequences
+
+### Positive
+
+By creating PFB inclusion proofs you create PFB fraud proofs at the same time.
+
+### Negative
+
+Optimization 1 might introduce more intra-transaction padding. Optimization 2 will introduce intra-transaction padding so only the PFB transaction occupies the shares.
```

Comment:
```suggestion
Optimization 1 would introduce more intra-transaction padding. Optimization 2 will introduce intra-transaction padding so only the PFB transaction occupies the shares.
```

Optimization 1 definitely introduces more intra-transaction padding because currently there is 0 intra-transaction padding.